config/default_records/submission_templates/012_scrna_core_library_prep_submission_templates.wip.yml

```diff
@@ -0,0 +1,40 @@
+# Submission templates for the aggregation and library prep parts of the scRNA Core pipeline.
```

Comment:
I think I also used 012 in the filename for ANOSPP (and other files) so we'll have to fix at merge into develop.